server/apis/v1/handler.go

```diff
@@ -88,8 +88,8 @@ func (h *handler) Login(c *gin.Context) {
 	// TODO - send a request to Dex to get the real user identity token.
 	token := "dummy-token"
 	c.SetCookie("user-identity-token", token, 3600, "/", "", true, true)
-	returnUrl := c.DefaultQuery("returnUrl", "/cluster-summary")
-	c.Redirect(http.StatusOK, returnUrl)
+	returnUrl := c.DefaultQuery("returnUrl", "/")
+	c.Redirect(http.StatusTemporaryRedirect, returnUrl)
```

Comment:
1. Should this redirect take the browser to dex server (not the return url)?
2. the http code may be a minor one. the dex doc uses http.StatusFound. https://dexidp.io/docs/using-dex/